flang/lib/Lower/Bridge.cpp

```diff
@@ -1482,6 +1483,18 @@ class FirConverter : public Fortran::lower::AbstractConverter {
       }
     }
 
+    if(ompSections){ 
```

Comment:
Could you add a comment on what the following code does and why it is necessary.
Nit: Could you also format?